CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ CPMAddPackage(
   VERSION 3.9.1
   URL https://github.com/nlohmann/json/releases/download/v3.9.1/include.zip
   URL_HASH SHA256=6bea5877b1541d353bd77bdfbdb2696333ae5ed8f9e8cc22df657192218cad91
+  PATCH_COMMAND
```

Comment:
Let's add a comment explaining why we're patching (Something like `# WAR compiler bug in nvcc 11.0, see NVIDIA/NVBench#18` would be fine).